.github/workflows/deploy-to-control-plane.yml

```diff
@@ -47,7 +46,19 @@ jobs:
         run: |
           cpln profile update default --token ${CPLN_TOKEN}
 
+      # Caching step
```

Comment:
@ahangarha @Judahmeek this caching might not be doing anything. Please research.